.kokoro/psm_interop_kokoro_lib.sh

```diff
@@ -0,0 +1,524 @@
+#!/usr/bin/env bash
+# Copyright 2020 gRPC authors.
```

Comment:
[Minor] Should be 2023 now

Reply:
That's the year the code was created, and this is 1:1 copy. We don't update the copyright in this case.